### PR TITLE
clean up capture_func_transforms flag

### PR DESCRIPTION
**Pull request by bobrenjc93**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #136960

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames @rec

### CODE REVIEW

**Review by guilhermeleobas — APPROVED** [review]

File: torch/_dynamo/config.py [hunk 1]
```diff
@@ -427,9 +427,6 @@ def default_debug_dir_root():
 
 capture_autograd_function = True
 
-# enable/disable dynamo tracing for `torch.func` transforms
-capture_func_transforms = True
```
Comment (anijain2305) [inline torch/_dynamo/config.py:431]: Dont delete the flag. Make it a no-op.

**Review by anijain2305 — DISMISSED** [review]

**Review by jansel — APPROVED** [review]

**Comment by jansel** [comment]
Should we add a comment to config.py that this flag is ignored?

**Pull request closed without merging**